help/destinations/catalog/advertising/acxiom-audience-distribution.md

```diff
@@ -0,0 +1,143 @@
+For the [!DNL Acxiom Audience Distribution] destination to correctly receive the audience data, you must map the source fields from Experience Platform to the correct [!DNL Acxiom Audience Distribution] target fields.
+
+[!DNL Acxiom Audience Distribution] only allows mapping with the following fields and requires the profiles that come through our system to be in the following order: 
+
+| Field Name         | Description|Required| Field Order | Max Length |
+|--------------------|------------|--------|-------------|------------|          
+|First Name          |First name of individual                                                                                                         |N|1           |255         |
+|Middle              |Middle name or initial of the individual                                                                                         |N|2           |50          |
+|Last Name           |Last name of the individual                                                                                                      |Y|3           |255         |
+|Generational Suffix |Suffix of the individual                                                                                                         |N|4            |10          |
+|Address Line 1      |Address 1 field of primary residence                                                                                             |Y|5            |255         |
+|Address Line 2      |Address 2 field of primary residence                                                                                             |N|6            |255         |
+|City                |City of primary residence                                                                                                        |Y|7           |255         |
+|State               |State abbreviation of primary residence                                                                                          |Y|8           |2           |
+|Zip Code            |Full zip code of primary residence                                                                                               |Y|9            |10          |
+|Email               |Primary email By default, this field is used as a deduplication key to make the records unique                                   |N|10         |255         |
+|Phone               |Phone number of individual (area code + number)<br> By default, this field is used as a deduplication key to make the records unique. |N|11          |10          |
+
+
+1. Under the **Source Field** column, in the red select source field boxes, enter the name of each of the attributes you want to map from your input file to the corresponding target field. If you do not know the attribute name, you can search for the attribute by selecting the **Select Arrow** icon and then selecting the source field from the directory on the **Select source field** screen.<br>

```

Comment:
You can remove the numbering in these two steps, since they are pretty easy to follow.